audio/encoding.py

```diff
@@ -38,6 +41,9 @@ def __init__(
         :param tuple(str)|None select_channels: tuple of (channel_layout, channel_name), see `ffmpeg -layouts`
             this is useful if the new encoding might have an effect on the duration, or if no duration was specified
             in the source corpus
+        :param bool|None recover_duration: This will open all files with "soundfile" and extract the length information.
+            There might be minimal differences when converting the encoding, so only set this to `False` if you're
+            willing to accept this risk.
```

Comment:
maybe add to the docstring explanation what happens if `None` is given (I can read the code but I don't understand why it is chosen that way)

> For all parameter holds that "None" means to use the ffmpeg defaults, which depend on the input file and the output format specified.

or is this also the case here?

Reply:
`None` does not refer to the `ffmpeg` default but the old job default. This is not about an `ffmpeg` option after all. I added a sentence on this, please see if that is sufficient from your pov.